tools/deploy_assisted_installer.py

```diff
@@ -65,6 +65,7 @@ def main():
             spec = data["spec"]["template"]["spec"]
             service_container = spec["containers"][0]
             service_container["env"].append({'name': 'DEPLOY_TARGET', 'value': "ocp"})
+            service_container["env"].append({'name': 'STORAGE', 'value': "filesystem"})
```

Comment:
Shouldn't this be "s3"? We use scality here, right?

Reply:
We only use s3 for the deployment_type_k8s right now. See: https://github.com/openshift/assisted-service/blob/master/cmd/main.go#L258-L267

Reply:
Ah okay :+1: